README.md

```diff
@@ -81,10 +81,76 @@ Finally, we look at the procedural steps adopted by the framework:
+- src/setup/deployment/raw-code/serverless/
+    - This directory contains the logic of the functions that are to be deployed.
+    - Each subdirectory contains source code resources for deploying to a specific provider (aws, azure, alibaba, gcr).
+    - The serverless.yml file is stored inside each provider's subdirectory.
+    - Function source code is stored in each provider’s subdirectories (e.g. aws/hellogo, azure/hellopy).
+
+
+### Serverless.com & Cloud Provider Capabilities
+While serverless.com framework is a powerful tool when it comes to deploying Lambda Functions to AWS, its capabilities are more limited with other providers. The following table shows serverless.com features vs. different providers whose deployment was considered to be automated.
+
+|                                              | AWS Lambda | Azure Function                           | Google Cloud Run | Knative | Google Cloud Function *                  | Alibaba                                  |
+|----------------------------------------------|------------|------------------------------------------|------------------|---------|------------------------------------------|------------------------------------------|
+| Deploy function - zip                        | Yes        | Yes                                      | No               | N/A     | Yes                                      | Yes                                      |
+| Deploy function - docker                     | Yes        | No (documentation does not mention it)** | No               | Yes     | No (documentation does not mention it)** | No (documentation does not mention it)** |
+| Python Runtime                               | Yes        | Yes                                      | No               | Yes     | Yes                                      | ??**                                     |
+| Go Runtime                                   | Yes        | No                                       | No               | Yes     | Yes                                      | ??**                                     |
+| Java Runtime                                 | Yes        | No                                       | No               | ??**    | No                                       | ??**                                     |
+| Node JS                                      | Yes        | Yes                                      | No               | Yes     | Yes                                      | Yes                                      |
+| HTTP trigger                                 | Yes        | Yes                                      | No               | Yes     | Yes                                      | Yes                                      |
+| Resource Management (e.g. buckets databases) | Yes        | Yes (at least it looks like it)          | No               | ??**    | ??**                                     | ??**                                     |
```

Comment:
We might need to change the readme a bit, but I think this is okay for now.
@kk-min Can you create a separate issue for this? So that we can attend to it later?

Reply:
I've created an issue here: #312